build/openresty/wasmx/wasmx_repositories.bzl

```diff
@@ -21,7 +21,7 @@ def wasmx_repositories():
         new_git_repository,
         name = "ngx_wasm_module",
         branch = ngx_wasm_module_branch,
-        remote = KONG_VAR.get("NGX_WASM_MODULE_REMOTE", "git@github.com:Kong/ngx_wasm_module.git"),
```

Comment:
Won't this impact the current wasmx team development / testing workflow - ie, how to easily test local changes of a nginx wasm module clone? We could leave it here for that, and add it to [this list](https://github.com/Kong/kong-ee/blob/d6387aeb6b4dcc551b5f2477f19ca871e581c261/build/kong_bindings.bzl#L98) so that a change in it causes the invalidation of kong_bindings (I think @hishamhm brought this invalidation issue up few days ago : )

Reply:
ahh, good call. I'll rework this tomorrow/later this week with that in mind.

Reply:
Yes we have many Bazel improvements we would like to make or else debugging ngx_wasm_module in the Gateway is going to be **extremely** impractical. All these needed updates are listed on page 3 of the integration document, if you're curious: https://docs.google.com/document/d/1Y-OL0QNb2CYVRfkEYuzLCc9dnbTVzb9mBiKL-fO7jUw/edit

Reply:
I've updated the pull request with the `NGX_WASM_MODULE_REMOTE` handling intact. The only change made here is switching the remote to http to be more inline with how we are fetching other git dependencies.